Gruntfile.js

```diff
@@ -4,9 +4,25 @@
 
 'use strict';
 
-var multiline = require('multiline');
+var multiline = require('multiline'),
+	xml2js = require('xml2js');
 
-module.exports = function(grunt) {
+var KABOB_REGEX = /\-(\w)/g;
+
+/**
+ * Transforms kabob case names to camel case
+ * @param name        ex: foo-bar-baz
+ * @returns {String}  ex: fooBarBaz
+ */
+function kabobToCamelCase( name ) {
```

Comment:
Let's skip the lodash util for now. If we end up using other lodash methods it'd make sense to pull the rest in. Updated spelling in 3299650